src/Tizen.NUI.Components/Controls/ItemsView/ItemSource/EmptySource.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace Tizen.NUI.Components
+{
+	sealed internal class EmptySource : IItemSource
```

Comment:
`IDE0036: Modifiers are not ordered`
```suggestion
	internal sealed class EmptySource : IItemSource
```